src/views/home/HomeViewProvider.ts

```diff
@@ -72,12 +79,39 @@ export class HomeViewProvider implements vscode.WebviewViewProvider {
         }
       }
     })
+
+    this.webviewIsDisposed = false
   }
 
   public revive(panel: vscode.WebviewView) {
     this._view = panel
   }
 
+  public async refreshAll() {
+    if (this.isRefreshing) {
+      return
+    }
+
+    this.isRefreshing = true
+
+    await vscode.window.withProgress(
+      {
+        location: { viewId: 'devcycle-home' },
+      },
+      async () => {
+        if (!this._view || this.webviewIsDisposed) {
+          return
+        }
+        this._view.webview.html = await this._getHtmlForWebview(this._view.webview)
+      }
+    )
+    this.isRefreshing = false
+  }
+
+  public async refresh() {
```

Comment:
Laura removed this method for the InspectorViewProvider in her [PR](https://github.com/DevCycleHQ/vscode-extension/pull/170/files#diff-084d91d4aa6b58d3c343d60f3c8aaaeed7b124c3d1e134d6aececb7a9a7e1f5aL140) cause it's redundant. I don't think it's needed for the HomeViewProvider either since all it does is call `refreshAll` which is public and can just be called directly

Reply:
ohh cool, that's true, added that just because the method was used in refresh all command. I'll rebase her branch, thanks!

Reply:
refresh all calls the `refreshAll` method of each provider so you should be good to make the change without having to rebase.

https://github.com/DevCycleHQ/vscode-extension/blob/main/src/commands/refreshAll/registerRefreshAllCommand.ts#L22

Reply:
hmmm...actually she just moved the method upwards, and the `refreshAll` and `refresh` still use the same `_refresh` method. The `refresh` method in the home view provider is still needed for `registerRefreshAllCommand`, so I think it's okay to leave it as is to be consistent with other providers?